client/cypress/integration/visualizations/box_plot_spec.js

```diff
@@ -38,6 +38,8 @@ const SQL = `
 `;
 
 describe('Box Plot', () => {
+  const viewportWidth = Cypress.config('viewportWidth');
```

Comment:
Cool :+1:

Reply:
I really didn't want to hardcode it :grin: